VocaDbModel/Domain/Users/IUser.cs

```diff
@@ -10,5 +10,21 @@ public interface IUser {
 
 		string Name { get; set; }
 
-	}
+	}
+
+	public static class IUserExtender {
+
+		public static bool IsTheSameUser(this IUser left, IUser right) {
+
+			ParamIs.NotNull(() => left);
+
+			if (right == null)
+				return false;
+
+			return left.Id == right.Id;
+
+		}

```

Comment:
Can I convert IsTheSameUser to an extension method like this?

Reply:
I think if you add IEquatable<IUser> to IUser then this should not be necessary.